src/LowerPattern.jl

```diff
@@ -1,11 +1,11 @@
 
 function assignments(assigned::ImmutableDict{Symbol, Symbol})
     # produce a list of assignments to be splatted into the caller
-    a = (:($patvar = $resultsym) for (patvar, resultsym) in assigned)
+    return (:($patvar = $resultsym) for (patvar, resultsym) in assigned)
 end
 
 # return the code needed for a pattern.
-function code(bound_pattern::BoundPattern, state::BinderState)
+function code(bound_pattern::BoundPattern, ::BinderState)
```

Comment:
It's common to just put the type annotation when the arg variable isn't needed in the method body